examples/event_handler_rest/sam/micro_function_template.yaml

```
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+    micro-function-example
+
+Globals:
+  Api:
+    TracingEnabled: true
+    Cors: # see CORS section
+      AllowOrigin: "'https://example.com'"
+      AllowHeaders: "'Content-Type,Authorization,X-Amz-Date'"
+      MaxAge: "'300'"
+    BinaryMediaTypes: # see Binary responses section
+      - "*~1*" # converts to */* for any binary type
+
+  Function:
+    Timeout: 5
+    Runtime: python3.10
+
+Resources:
+    # Lambda Function Solely For /users endpoint
+    AllUsersFunction:
+        Type: AWS::Serverless::Function
+        Properties:
+            Handler: app.lambda_handler
+            CodeUri: users
+            Description: Function for /users endpoint
+            Architectures:
+                - x86_64
+            Tracing: Active
+            Events:
+                UsersPath:
+                    Type: Api
+                    Properties:
+                        Path: /users
+                        Method: GET
+            MemorySize: 512 # This Lambda Function can have a higher memory size set
```

```suggestion
            MemorySize: 128 # This Lambda Function can have a higher memory size set
```

It's better to keep 128 MB to avoid unexpected costs if someone wants to try this with many requests.